src/mumble_exe/mumble_exe.cpp

```diff
@@ -125,8 +137,151 @@ static const std::wstring GetAbsoluteMumbleAppDllPath(std::wstring suggested_bas
 	return base_dir + L"\\mumble_app.dll";
 }
 
+// UIAccessDisabledViaConfig queries the configuration store, and returns
+// whether UIAccess has been disabled by the user via the "shortcuts/uiaccess"
+// config option.
+static bool UIAccessDisabledViaConfig() {
+	// RegQueryValueEx only zero-terminates if the registry value's type is
+	// one of the Windows registry's string types. Because of this, we have
+	// to use a buffer of 7 elements in order to query for the string "false".
+	//
+	// If the value's type is string, we'll end up with "false\0\0", because we
+	// always zero-pad.
+	//
+	// But that's better than ending up in an infinite loop because we failed
+	// to zero-terminate.
+	wchar_t buf[7];
+	memset(&buf, 0, sizeof(buf));
+	DWORD sz = sizeof(buf) - 1*sizeof(wchar_t);
+
+	HKEY key = NULL;
+	bool success = (RegOpenKeyExW(HKEY_CURRENT_USER, L"Software\\Mumble\\Mumble\\shortcuts\\windows\\uiaccess", NULL, KEY_READ, &key) == ERROR_SUCCESS) &&
+	               (RegQueryValueExW(key, L"enable" , NULL, NULL, (LPBYTE)&buf, &sz) == ERROR_SUCCESS);
+	if (success && _wcsicmp(buf, L"false") == 0) {
+		return true;
+	}
+
+	return false;
+}
+
+// SystemSupportsUIAccess returns true if the
+// operating system supports UIAccess.
+static bool SystemSupportsUIAccess() {
+	OSVERSIONINFOEXW ovi;
+	memset(&ovi, 0, sizeof(ovi));
+
+	ovi.dwOSVersionInfoSize = sizeof(ovi);
+	GetVersionEx(reinterpret_cast<OSVERSIONINFOW *>(&ovi));
```

Comment:
```GetVersionEx``` is deprecated (which was a crap move by MS as there is no equivalent replacement :( ). Unless that isn't available to us use ```IsWindowsVistaOrGreater``` from versionhelpers.h (https://msdn.microsoft.com/en-us/library/windows/desktop/dn424972(v=vs.85).aspx) instead.